src/common/components/nav/desktopNav.tpl.html

```diff
@@ -0,0 +1,124 @@
+<header id="desktop-navigation" class="gh-desk-nav">
+  <div class="container">
+
+    <a href="" title="Cru" class="gh-desk-logo"></a>
+
+    <ul class="gh-desk-nav-tree col-level-1">
+      <li ng-repeat="i in $ctrl.menuStructure.main" ng-class="{'has-children': i.children.length, 'open-menu': $ctrl.mobileMenuPath.main[0] === i.path.split('/').pop()}"
```

Comment:
I can make the variable name longer, but I kinda like having it just be re-used in the loop.  Maybe something like menuItem. Sound good?